javascript/workflow/src/resources/DataPackage/ResourceManager.ts

```diff
@@ -249,11 +249,11 @@ export class ResourceManager {
 	 * @param handler - The profile handler to register
 	 */
 	public registerProfile(handler: ProfileHandler): void {
-		if (this._profileHandlers.has(handler.profile)) {
-			console.warn(
-				`A resource handler for profile '${handler.profile}' is already registered. Overriding existing entry.`,
-			)
-		}
+		// if (this._profileHandlers.has(handler.profile)) {
```

Comment:
Let's add default profiles to the constructor so we can leave the warning

Reply:
Oops, I did not mean to commit that! Will do the constructor update...